py/server/deephaven/__init__.py

```diff
@@ -25,3 +26,7 @@
 from .replay import TableReplayer
 from ._gc import garbage_collect
 from .dbc import read_sql
+
+__all__ = ["read_csv", "write_csv", "kafka_consumer", "kafka_producer", "empty_table", "time_table", "merge",
+           "merge_sorted", "new_table", "input_table", "DynamicTableWriter", "TableReplayer", "garbage_collect",
+           "read_sql", "DHError", "SortDirection"]
```

Comment:
I'm not sure what my thought is, but we are not importing any parquet stuff.  This is probably ok, but I'm noting it.